bin/uglifyjs

```diff
@@ -575,6 +581,15 @@ function read_whole_file(filename, cb) {
     }
 }
 
+function read_source_map(code) {
+    var match = /\n\/\/# sourceMappingURL=data:application\/json(;.*?)?;base64,(.*)/.exec(code);
+    if (!match) {
+        print_error("ERROR: can't read source map");
```

Comment:
Perhaps add "inline" to the message so it's easier to catch configuration errors (I'm thinking along the lines of confusing `--source-map-inline` with `--in-source-map inline`).